### PR TITLE
Format code with black

### LINKED ISSUE #12: Integrate / evaluate black
We should evaluate if [black](https://github.com/ambv/black) is ready for being the default formatter in our DRF projects.

### PR DESCRIPTION
**Pull request by sliverc**
Fixes #12 
Fixes #65 

Using `importlib.import_module` to make code valid python code before generation so it can be formatted with black.

Black requires Python 3.6 so bumped version. No code changes were needed.

### CODE REVIEW

File: {{cookiecutter.project_name}}/.flake8 [hunk 1]
```diff
@@ -2,33 +2,25 @@
...
+    E266,
+    # line too long (managed by black)
+    E501,
+    # Line break occurred before a binary operator (this is not PEP8 compatible)
+    W503,
+    # do not enforce existence of docstrings
     D100,
-    # Missing docstring in public class
     D101,
-    # Missing docstring in public method
```
Comment (winged) [inline {{cookiecutter.project_name}}/.flake8]: Why are we removing the comments here? As long as the ignore codes are here, I suggest keeping the comments as well

File: {{cookiecutter.project_name}}/{{cookiecutter.project_name}}/{{cookiecutter.django_app}}/migrations/0001_initial.py [hunk 2]
```diff
@@ -12,35 +12,119 @@ class Migration(migrations.Migration):
...
-                ('email', models.EmailField(blank=True, max_length=254, verbose_name='email address')),
-                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
-                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
-                ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
-                ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.Group', verbose_name='groups')),
-                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.Permission', verbose_name='user permissions')),
+                (
```
Comment (winged) [inline {{cookiecutter.project_name}}/{{cookiecutter.project_name}}/{{cookiecutter.django_app}}/migrations/0001_initial.py]: Didn't we once say we'd exclude migrations from autoformatting?

Comment (sliverc) [reply {{cookiecutter.project_name}}/{{cookiecutter.project_name}}/{{cookiecutter.django_app}}/migrations/0001_initial.py]: Not sure that we said this. But this is much more readable and more consistent this way. Especially as in migrations their might also be custom code which certainly needs to be formatted.

File: {{cookiecutter.project_name}}/.flake8 (hunk 1 above)
Comment (sliverc) [reply {{cookiecutter.project_name}}/.flake8]: That was a copy pasted .flake8 file... :wink: added again.

File: {{cookiecutter.project_name}}/{{cookiecutter.project_name}}/{{cookiecutter.django_app}}/migrations/0001_initial.py (hunk 2 above)
Comment (winged) [reply {{cookiecutter.project_name}}/{{cookiecutter.project_name}}/{{cookiecutter.django_app}}/migrations/0001_initial.py]: Just wasn't sure anymore. Let's keep it autoformatted then :)

**Pull request merged**